### ISSUE TITLE
[Melhoria] no [DICT] - Paginação na listagem de reivindicações

### ISSUE THREAD

**Issue by vtlino** [bacen/pix-perguntas-e-respostas#217]
Poderia ter uma paginação na listagem de reivindicações, uma vez que o limite é 200. Hoje não vejo uma maneira inteligente de fazer uma busca em um intervalo curto com muitas reivindicações.

**Comment by luizlaydner** [comment]
Está no nosso backlog fazer uma melhoria nos endpoints de listagem.

A ideia é criar novos parâmetros (`afterId` e `beforeId`) que servirão como cursores.
Tendo feito uma 1a requisição, você pegaria o último Id da lista e na próxima requisição o incluiria como parâmetro `afterId`. 
Com o cursor, será possível iterar na lista sem ter que lidar com elementos repetidos, quando se itera com intervalos de tempos "fechados".
Resolveria também a situação de haver mais de 200 reivindicações com o mesmo timestamp e não ser possível listar todas.
Essa melhoria seria provavelmente pra depois de 5/10.

Isso atenderia? Alguma sugestão ?

**Comment by vtlino** [comment]
Com a criação desses novos parâmetros (mantendo o ModifiedAfter e ModifiedBefore) resolveria sim.

Obrigado.

**Comment by vtlino** [comment]
@luizlaydner, essa mesma melhoria será feita para Infração?

**Comment by luizlaydner** [comment]
Sim. A intenção é fazer a melhoria para todos os endpoints de listagem.
